server/health/health.go

```diff
@@ -0,0 +1,25 @@
+package health
+
+import (
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/health"
+	"google.golang.org/grpc/health/grpc_health_v1"
+)
+
+const (
+	ServiceName = "proto.API" // taken from compiled protobuf file api.go.pb (line 793)
```

Comment:
Does this need to be exported if it's not used outside the package?

Reply:
Nope, it doesn't – I unexported it.